yb-voyager/src/tgtdb/main_test.go

```diff
@@ -124,12 +136,47 @@ func TestMain(m *testing.M) {
 		}),
 	}
 
+	// 3. Create a yugabytedb single node container
 	err = testYugabyteDBTarget.Init()
 	if err != nil {
 		utils.ErrExit("Failed to connect to yugabytedb database: %w", err)
 	}
 	defer testYugabyteDBTarget.Finalize()
 
+	// 4. Create a yugabytedb 3 node cluster container
+	yugabytedbClusterContainer := testcontainers.NewYugabyteDBCluster(&testcontainers.ContainerConfig{
+		DBType:            testcontainers.YUGABYTEDB,
+		NodeCount:         3,
+		ReplicationFactor: 3,
+	})
+	err = yugabytedbClusterContainer.Start(ctx)
+	if err != nil {
+		utils.ErrExit("Failed to start yugabytedb cluster: %v", err)
+	}
+	host, port, err = yugabytedbClusterContainer.GetHostPort()
+	if err != nil {
+		utils.ErrExit("%v", err)
+	}
+	testYugabyteDBTargetCluster = &TestTargetYugabyteDBCluster{
+		YugabyteDBClusterContainer: yugabytedbClusterContainer,
+		TargetYugabyteDB: newTargetYugabyteDB(&TargetConf{
+			TargetDBType: "yugabytedb",
```

Comment:
Nit: Can we use the YUGABYTEDB constant here. Similarly in other TargetDBTypes above?